Source/JavaScriptCore/jit/JITCall.cpp

```diff
@@ -270,21 +267,23 @@ void JIT::compileOpCall(const JSInstruction* instruction, unsigned callLinkInfoI
     emitGetVirtualRegister(callee, BaselineJITRegisters::Call::calleeJSR);
     storeValue(BaselineJITRegisters::Call::calleeJSR, Address(stackPointerRegister, CallFrameSlot::callee * static_cast<int>(sizeof(Register)) - sizeof(CallerFrameAndPC)));
 
-    if (compileCallDirectEval(bytecode))
+    if (opcodeID == op_call_direct_eval) {
```

Comment:
This is unrelated, right?

Reply:
Cleaning up the flow to make it explicit that op_call_direct_eval does not use CallLinkInfo. (since we return in this branch)